<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>select_t</class>
 <widget class="QDialog" name="select_t">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tests</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>100</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 1px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
       </property>
       <property name="title">
        <string>EPS INFORMATION</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>CHARGER ASSY.PT.NO</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>BATTERY PACK.PT.NO</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>270</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>550</width>
         <height>270</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">

QGroupBox {
    border: 1px solid gray;
    border-radius: 1px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}

</string>
       </property>
       <property name="title">
        <string>Select Tests</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="vtest">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>45</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Battery Charger tests</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="Insulation_t">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Insulation Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Bonding_t">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Bonding Test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Continuity_t">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>continuity test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Operating_logic_t">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Operating Logic test</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Fast_c_termi">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Fast charge &amp; charge termi</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="charge_inhibit">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Charge inhibition</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="self_test_fun">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Self test function</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="output">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Output</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,0,0,0,0,0,0">
          <property name="spacing">
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Battery Pack Assembly</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>10</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="preliminary_op">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Preliminary Operation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="charge">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Charge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Discharge">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Discharge</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Serviceability">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Serviceability</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Conditioning">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>17</height>
             </size>
            </property>
            <property name="text">
             <string>Conditioning</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder></zorder>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>select_t</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>select_t</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
